tests/test_cli.py

```diff
@@ -69,7 +69,7 @@ def test_wrong_usage(self):
         c = Cli(None)
         # Check if all commands are mentioned in their respective usage texts
         with pytest.raises(AttributeError):
-            assert getattr(c, "foobar_usage")() == False
+            assert not getattr(c, "foobar_usage")()
```

Comment:
I completely don't understand this test. The comment says something about commands and their respective usage texts but I don't see any check of the usage text. Also, if the call to `getattr` should raise the AttributeError, there is no need to try to call the returned value and negate its return value and check it with an assert.

If this test is useful, it should look like this:
```suggestion
            getattr(c, "foobar_usage")
```
This raises `AttributeError: 'Cli' object has no attribute 'foobar_usage'` so nothing more should be needed.